source/vocab/display.jsonld

```diff
@@ -120,6 +126,12 @@
           "classLensDomain": "Identifier",
           "showProperties": [ "value", "typeNote", "marc:hiddenValue" ]
         },
+        "ISBN": {
+          "@id": "ISBN-chips",
+          "@type": "fresnel:Lens",
+          "classLensDomain": "ISBN",
+          "showProperties": [ "value", "qualifier" ]
+        },
```

Comment:
Could be better to just add qualifier to Identifier chips.

Reply:
I went back and forth with this one for a while, since there are in fact several Identifiers with qualifier. For now I chose to try this one out, First of all I prefer if we don't make the chips longer than we have to, and this one is so explicit in praxis. After studying the almighty shapes I felt qualifier in the other Identifiers witness more about noise and data which might have ended up in the wrong place. I know this can also be of use but I think we need to revisit Identifiers more properly (in the future!) and see this as a test and see if the feedback is good or not.

Reply:
Sounds good! I agree then that this captures a meaningful distinction (and singles out qualifier for ISBN as something we could address in isolation (since it is reasonably redundant if the ISBN is on the correctly described instance...)).